src/viur/core/render/vi/__init__.py

```diff
@@ -87,7 +87,7 @@ def dumpConfig():
         "modules": res,
         # "configuration": dict(conf.admin.items()), # TODO: this could be the short vision, if we use underscores
         "configuration": {
-            k.replace("_", "."): v for k, v in conf.admin.items()
+            {f"admin.{key}": values for key, values in conf.admin.items()}
```

Comment:
That's now a dict nested in a set and `"admin.color_secondary"` instead of `"admin.color.secondary"`


```suggestion
            k.replace("_", "."): v for k, v in conf.admin.items(True)
```